interface/main/messages/save.php

```diff
@@ -109,10 +111,10 @@
 
             $runQuery ="select * from facility order by name";
             $fetch = sqlStatement($runQuery);
-            while ($frow = sqlFetchArray($fetch)) { $facilities[] = $frow; }
+            while ( $frow = sqlFetchArray( $fetch ) ) { $facilities[] = $frow['id']; }
```

Comment:
Ouch.  Guess this doesn't happen much.  Is there a way to do full end-to-end validation via their API to make sure all the pieces work?